docs/core/versions/index.md

```diff
@@ -105,24 +105,26 @@ A new version of .NET Core could mean several things:
 
 ### Shipping a patch release
 
-After shipping a .NET Core v1.0.0 stable version, patch-level changes (no new APIs) are made to .NET Core libraries to fix bugs and improve performance and reliability. The various metapackages are updated to reference the updated .NET Core library packages. The metapackages are versioned as patch updates (x.y.z). Frameworks are not updated. A new .NET Core distribution is released with a matching version number to the `Microsoft.NETCore.App` metapackage.
+Patch-level changes (no new APIs) are made to .NET Core libraries to fix bugs and improve performance and reliability. The various metapackages are updated to reference the updated .NET Core library packages. The metapackages are versioned as patch updates (x.y.z). Frameworks are not updated. A new .NET Core distribution is released with a matching version number to the `Microsoft.NETCore.App` metapackage.
 
-You can see patch updates demonstrated in the project.json examples below.
+You can see patch updates demonstrated in the metapackage references of the following project file:
 
-```
-{
-  "dependencies": {
-    "Microsoft.NETCore.App": "1.0.1"
-  },
-  "frameworks": {
-    "netcoreapp1.0": {}
-  }
-}
+```xml
+<Project Sdk="Microsoft.NET.Sdk.Web">
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp1.1</TargetFramework>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="Microsoft.AspNetCore.Server.Kestrel" Version="1.1.1" />
+    <PackageReference Include="Microsoft.AspNetCore.Mvc" Version="1.1.2" />
+  </ItemGroup>
+</Project>
```

Comment:
This whole page describes versioning of .Net Core, but here you're showing ASP.NET Core, which is technically not part of .Net Core and so I think is not really relevant here.

On the other hand, the two metapackages (`NETStandard.Library` and `Microsoft.NETCore.App`) are not normally visible in a csproj. Maybe remove the example and describe how does msbuild decide which patch version of a metapackage to use?